server/tests/services/user.test.js

```diff
@@ -489,6 +489,16 @@ describe('## User Service', () => {
           done();
         });
     });
+
+    it('should fail when the same group is added twice', (done) => {
```

Comment:
nit: it's not failing when the same group is added, it's just ignoring the second group